### DISCUSSION TITLE
Should result blocks show in the pnp result web part if query rules are allowed?

### DISCUSSION THREAD

**Discussion by marting1010** [microsoft-search/pnp-modern-search#1594] [General]
I can get Promoted results working in the PnP Results web part, but I can't seem to get Result Blocks working. Basically, I have a search vertical working against LocalSharepointResults with a result block to search for any people with "SQL Server" in their Skills people property (which looks at Local People Results. It doesn't seem to work though in the Results Web Part.

**Comment by wobba** [comment]
Pick the debug template  and you will see a block `secondaryResults` returned which you can iterate over.

Similar to the v3 docs
https://microsoft-search.github.io/pnp-modern-search/v3/search-parts/templating/#accessing-items

```
{{#each data.secondaryResults as |item|}}
```
